### ISSUE TITLE
Loading Screen Crash (auto-install bash)

### ISSUE THREAD

**Issue by JayChen078** [JulienMaille/dribbblish-dynamic-theme#177]
**Describe the bug**
Auto-install from, curl -fsSL https://raw.githubusercontent.com/JulienMaille/dribbblish-dynamic-theme/master/install.sh | sh, ends up with a loading screen that crashes "starting up" or "one moment getting ready". 

**To Reproduce**
1. curl -fsSL https://raw.githubusercontent.com/JulienMaille/dribbblish-dynamic-theme/master/install.sh | sh

**Comment by JayChen078** [comment]
warning Config "extensions" unchanged: default-dynamic.js is not on the list.
warning Config "extensions" unchanged: dribbblish.js is not on the list.
warning Config "extensions" unchanged: Vibrant.min.js is not on the list.

This might be the issues, where are these .js files located?

**Comment by JulienMaille** [comment]
No this is part of some cleanup, not an issue if these files do not exist. Can you post the complete log?

**Comment by JayChen078** [comment]
Issue fixed.

**Comment by JulienMaille** [comment]
How did you fix it?

**Comment by JayChen078** [comment]
I did a clean reinstall of everything - removed any and all data belonging to spotify and spicetify. Upon reinstallation it worked - although there might be a few times it might crash again here and there and I'd have to /spicetify apply, and it would fix itself.

**Comment by JayChen078** [comment]
@JulienMaille 

Actually, try this, I noticed that this allowed spicetify marketplace and DribbblishDynamic to run together. 

Delete all your data, belonging to spicetify and spotify, and completely reinstall them. Start from scratch. 

From a clean spicetify config, I downloaded the Dribbblish (default theme), found here. https://github.com/morpheusthewhite/spicetify-themes/tree/master/Dribbblish. Then drag it into your themes folder, and run: 

spicetify config extensions dribbblish.js
spicetify config current_theme Dribbblish color_scheme base
spicetify config inject_css 1 replace_colors 1 overwrite_assets 1
spicetify apply

Spotify should launch and be fully functional under just the default Dribbblish theme. Now, without touching anything else, I installed spicetify marketplace, using curl & auto install. 

curl -fsSL https://raw.githubusercontent.com/CharlieS1103/spicetify-marketplace/main/install.sh | sh
spicetify apply

At this stage, spotify should launch in the default Dribbblish theme and spicetify marketplace should work without crashes. If you did a clean reinstall, spicetify marketplace should be blank, no previous history of installs whatsoever. Then through spicetify marketplace, I was able to just simply click and download DribbblishDynamic, and reload the page for it to work. 

Currently I have:

MacOS  v 12.2.0
Spotify v 1.1.79.763
Spicetify v 2.9.1
Dribbblish v Beta-ba5846b

I'm not entirely sure why, this process works, but I assume it might be related to the Dribbblish.js file installed from the default Dribbblish theme. It seems that DribbblishDynamic tends to crash on marketplace without installing the Dribbblish.js file in extensions. In my config, under extensions, I have Dribbblish.js and under current_theme I have Dribbblish still - this seems to fix the problem, let me know if this works or doesn't work for you.
